src/components/header/Logo.tsx

```diff
@@ -0,0 +1,12 @@
+import logo from "../../assets/images/logo.png";
+import "../../styles/header/header.scss";
+
+const Logo = () => {
+  return (
+    <div>
+      <img className="headerLogo" src={logo} alt="" />
```

Comment:
classNames should be consistent, either camel case or dash seperated it doesn't matter